app/services/record_declaration.rb

```diff
@@ -101,6 +101,7 @@ def declaration_attempt
         declaration_type:,
         cpd_lead_provider:,
         user:,
+        evidence_held:,
```

Comment:
could you please add a test to record declaration to capture we're saving this on both the attempt and the declaration?

Reply:
All done tks for your review @cwrw @mooktakim 👍 😄